lisa/src/main/java/it/unive/lisa/cfg/statement/NativeCall.java

```diff
@@ -45,7 +61,27 @@ protected NativeCall(CFG cfg, String constructName, Expression... parameters) {
 	 * @param parameters    the parameters of this call
 	 */
 	protected NativeCall(CFG cfg, String sourceFile, int line, int col, String constructName, Expression... parameters) {
-		super(cfg, sourceFile, line, col, parameters);
+		super(cfg, sourceFile, line, col, Untyped.INSTANCE, parameters);
```

Comment:
Change this whole constructor with `this(cfg, sourceFile, line, col, constructName, Untyped.INSTANCE, parameters)`